autonomy/configurations/loader.py

```diff
@@ -74,10 +74,24 @@ def load_service_config(
     service_config = apply_env_variables(
         service_config, env_variables=os.environ.copy()
     )
+
+    if "dependencies" in service_config:
+        dependencies = load_dependencies(
+            dependencies=service_config.pop("dependencies")
+        )
+    else:
+        dependencies = {}
+        warn(
+            "`dependencies` parameter not defined in the service",
+            FutureWarning,
+            stacklevel=2,
+        )
+        print("WARNING: `dependencies` parameter not defined in the service")
```

Comment:
Warn if `dependencies` param is missing from the service config. This will be optional for this release, from the next release this parameter will be mandatory.